bucket_11/darktable/manifests/plist.docs

```diff
@@ -0,0 +1,4 @@
+share/doc/darktable/AUTHORS
+share/doc/darktable/LICENSE
```

Comment:
the LICENSE file is duplicate (since the LICENSE block is defined).
Best practice is to remove the LICENSE in a post-install step (or patch makefile to not install it the first place)
Not a dealbreaker but I typically remove these duplicates.